index.html

```diff
@@ -8,23 +8,33 @@
     <title>Tom full-stack web developer</title>
   </head>
   <body>
-    <header>
+    <header id="main">
       <div class="nav-bar">
         <div class="for-logo">
-          <a class="logo" href="#">Tom Daniel</a>
+          <a class="logo" href="#">Tom</a>
         </div>
         <nav class="nav-link">
           <div class="Hambuger">
+            <div id="hamma">
             <span class="bar"></span>
             <span class="bar"></span>
             <span class="bar"></span>
+            </div>
             <ul id="navLinks" class="navbar-list">
-              <li class="desktop-nav-list"><a href="#works-1">Portfolio</a></li>
+              <li class="desktop-nav-list"><a href="#main">Portfolio</a></li>
               <li class="desktop-nav-list"><a href="#about">About</a></li>
               <li class="desktop-nav-list"><a href="#contact">Contact</a></li>
             </ul>
           </div>
         </nav>
+        <nav class="mobile-nav">
+          <span class="close-btn">x</span>
+          <ul class="mobile-menu">
+            <li class="nav-list"><a href="#main">Portfolio</a></li>
```

Comment:
- [OPTIONAL] The 'Portfolio' link would be better to redirect the user to the works section instead of the first section.